python-package/setup.py

```diff
@@ -20,6 +22,22 @@ def read(filename):
     return io.open(os.path.join(CURRENT_DIR, filename), encoding='utf-8').read()
 
 
+def copy_files():
+
+    def copy_files_helper(folder_name):
+        src = os.path.join(CURRENT_DIR, os.path.pardir, folder_name)
+        if os.path.exists(src):
```

Comment:
Is `os.path.isdir` better?

Reply:
Just out of curiosity, why is it better?